src/buildNET6.xml

```diff
@@ -1,25 +0,0 @@
-<Project
-    xmlns="http://schemas.microsoft.com/developer/msbuild/2003"
-    ToolsVersion="15.0"
-    DefaultTargets="Build">
-<PropertyGroup>
-    <SolutionNET6>Dynamo.All.Net6.sln</SolutionNET6>
-    <NuGetPath>$(MSBuildProjectDirectory)\..\tools\Nuget\NuGet.exe</NuGetPath>
- </PropertyGroup>
-	
-<ItemGroup>
-    <ProjectToBuild Include="$(SolutionNET6)" >
-        <Properties>Configuration=Release;Platform=NET60_Windows</Properties>
```

Comment:
I think removing this file may break the master-15 builds... @QilongTang does that sound right?

Reply:
Master 15 will probably need to implement some conditionals based on the branch name
if master => no more net6 solutions or config
if 2,19 branch => as before

Reply:
Removing the file may break but replacing is probably fine. The only part concerning me is that the build path is totally different from before, this change does not worry me for now